pages/linux/qmrestore.md

```diff
@@ -1,8 +1,20 @@
 # qmrestore
 
-> Restore QemuServer vzdump Backups.
+> Restore QemuServer vzdump backups.
 > More information: <https://pve.proxmox.com/pve-docs/qmrestore.1.html>.
 
-- Restore KVM-based virtual machine to local storage:
+- Restore virtual machine from given backup file on the original storage:
 
-`qmrestore {{/var/lib/vz/dump/backup_file.vma.lzo}} {{vm_id}} --storage {{local}}`
+`qmrestore {{path/to/vzdump-qemu-100-2022_10_07-09_10_10.vma.lzo}} {{100}}`
+
+- Overwrite existing virtual machine from given backup file on the original storage:
+
+`qmrestore {{path/to/vzdump-qemu-100-2022_10_07-09_10_10.vma.lzo}} {{100}} --force true`
+
+- Restore virtual machine from given backup file on a specific [storage]:
```

Comment:
```suggestion
- Restore virtual machine from a given backup file on a specific [storage]:
```

Reply:
So the mnemonics part is only valid for single-character flags?

Reply:
> So the mnemonics part is only valid for single-character flags?

Yeah